README.md

```diff
@@ -1,3 +1,128 @@
-# Cartoomics-Grant
-![Screen Shot 2022-08-19 at 10 34 46 AM](https://user-images.githubusercontent.com/70932395/185665797-f3f3ccc0-c070-45bb-ae7e-e6d8acf5b664.png)
-<img width="824" alt="Screen Shot 2022-08-19 at 9 43 43 AM" src="https://user-images.githubusercontent.com/70932395/185656682-fff53d5c-eecf-4b0d-a259-2fa260385ba5.png">
+# Cartoomics Figure Generation 
+
+This Cartoomics repository consists of a workflow written in python that can generate an interaction network using a given knowledge graph from a user derived description of a cartoon image. The goal of this workflow is to generate complete and detailed pathway diagrams using the vast information contained within a knowledge graph.
+
+## Getting Started
+
+These instructions will provide the necessary environments, programs with installation instructions, and input files in order to run the creating_subgraph_using_cosinesim.py script, which will run this workflow from end to end.
+
+The path search algorithms available are:
+
+- Shortest Path Search: find the shortest path between each source and target node (may not be identical each time)
+- Cosine similarity Prioritization: prioritize all shortest paths between each source and target node by maximizing total cosine similarity of the start and all intermediate nodes to the target node in each path
```

Comment:
I think we'll need to include more detailed explanations of these methods. Potentially a future wiki. These are good for now.

Reply:
yes agreed! Can be added to issues